Lesson04/HW04.Birthday/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HW04.Birthday
+{
+    internal class Program
+    {
+        public int year, month, day;
+        public void info()
+        {
+            Console.Write("Год = ");
+            year = int.Parse(Console.ReadLine());
+            Console.Write("Месяц = ");
+            month = int.Parse(Console.ReadLine());
+            Console.Write("День = ");
+            day = int.Parse(Console.ReadLine());
+        }
+        public int year_2, month_2, day_2;
+        public void info_t()
+        {
+            Console.Write("Год = ");
+            year_2 = int.Parse(Console.ReadLine());
+            Console.Write("Месяц = ");
+            month_2 = int.Parse(Console.ReadLine());
+            Console.Write("День = ");
+            day_2 = int.Parse(Console.ReadLine());
+        }
```

Comment:
Names must not contain underscores or numbers. Read about variable naming

Reply:
Done